OperatingSystem-Services/Service-SOC/BasicDocumentation/DockerInstall-Wazuh.md

```diff
@@ -0,0 +1,48 @@
+# How to install Docker For Wazuh Purposes
+
```

Comment:
A) You can say this guide was created by you. Two reasons. 1) Clout and 2) So we know who to ask later questions. 

But generally, you want to have a summary page to explain what the guide is for. The header can be a shorthand for what you're doing, but you'll need to expand it.